internal/gatewayapi/testdata/backendtrafficpolicy-with-loadbalancer.in.yaml

```diff
@@ -91,3 +110,33 @@ backendTrafficPolicies:
       type: ConsistentHash
       consistentHash:
         type: SourceIP
+- apiVersion: gateway.envoyproxy.io/v1alpha1
```

Comment:
instead of creating a new route, can we just add this `slowStart` config to the existing config ?

Reply:
The existing route has been used to test `ConsistentHash`, so a new route needs to be added